source/agora/flash/Channel.d

```diff
@@ -1842,10 +1842,14 @@ LOuter: while (1)
         if (error.payment_hash in this.payment_hashes ||
             error.payment_hash in this.dropped_htlcs)
         {
+            const shared_secret = error.payment_hash in this.payment_hashes ?
+                                    this.payment_hashes[error.payment_hash] :
+                                    this.dropped_htlcs[error.payment_hash].shared_secret;
```

Comment:
Is it guaranteed the hash will be in `dropped_htlcs`?

Reply:
Yep, the if above guarantees that.